src/pages/iframes/comments/[dtpId].tsx

```diff
@@ -1,10 +1,10 @@
 import { GetServerSideProps, NextPage } from "next";
 import Error from "next/error";
 import * as React from "react";
-import styled from "styled-components";
 
 import { CommentInput } from "../../../components/comment-input";
 import { CommentList } from "../../../components/сomment-list";
```

Comment:
Turns out we had Cyrillic ‘с’ instead of Latin ‘c’ here 🙃

Spotted via https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

<img width="560" alt="Screenshot 2021-12-14 at 10 18 59" src="https://user-images.githubusercontent.com/608862/145979712-e5d3eb4a-8f42-4b04-b7ae-91afc93946dd.png">

This is unrelated to the Netlify bug though.

Reply:
Ups, sorry. I'm using the same extension.